haskell/lint.bzl

```diff
@@ -31,7 +30,7 @@ def _haskell_lint_rule_impl(ctx):
 def _haskell_lint_aspect_impl(target, ctx):
     hs = haskell_context(ctx, ctx.rule.attr)
 
-    if HaskellInfo not in target or HaskellPrebuiltPackageInfo in target:
```

Comment:
What will happen for the prebuilt dependencies here? Are they still ignored by the lint?